eng/scripts/Test-Code.ps1

```diff
@@ -93,9 +163,15 @@ try {
     $resultsArg = "--results-directory '$TestResultsPath'"
     $loggerArg = "--logger 'trx'"
 
-    Invoke-LoggedCommand `
-        -Command "dotnet test $coverageArg $resultsArg $loggerArg" `
-        -AllowedExitCodes @(0, 1)
+    if ($UseNative) {
+        Invoke-LoggedCommand `
+            -Command "dotnet test $coverageArg $resultsArg $loggerArg --no-build" `
+            -AllowedExitCodes @(0, 1)
+    } else {
+        Invoke-LoggedCommand `
+            -Command "dotnet test $coverageArg $resultsArg $loggerArg" `
```

Comment:
Might be cleaner to create a command string and append "--no-build" if it is testing Native Compilation rather than using the same logic twice.

Reply:
agree, updated.